data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>First sentence:</p><p> </p><p>Original:</p><p>Next, count the number of trips that have been taken from each station per day.</p><p> </p><p>Proposed:</p><p>Next, count the number of <strong>bikes</strong> that have been taken from each station per day.</p><p> </p><p>Or:</p><p>Next, count the number of trips that have <strong>started</strong> from each station per day.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done (Option 2) </p>